freqtrade/exchange/exchange.py

```diff
@@ -228,6 +228,18 @@ def get_quote_currencies(self) -> List[str]:
         markets = self.markets
         return sorted(set([x['quote'] for _, x in markets.items()]))
 
+    def get_pair_quote_currency(self, pair: str) -> str:
```

Comment:
This can return None, maybe annotate the returned value as `Optional[str]`? to have a tip for proper handling in places where it's used and the returned value is None i.o. str? Or, maybe it's better to return `''` here as the default value for `.get()`?

Reply:
I think returning an empty string is better - None forces you to explicitly none-check afterwards - which can lead to very awkward code / comparisons.

Reply:
I guess actually it should through an exception to handle properly the situation at the higher level when markets are not available/cannot be updated, handling may differ for different places where this info is used, but that's not for this PR and is quite complex, and returning empty string for another situation, when a pair cannot be found in the up-to-date market data is probably better than None, yes, since it avoids misusing of the None value when a string expected... So this LGTM 👍